uefi.adoc

```diff
@@ -1,37 +1,52 @@
 [[uefi]]
 == UEFI Requirements
-=== UEFI Version
-=== UEFI Compliance
-=== UEFI Platform Initialization
-The RISC-V Boot and Runtime Services Specification does not require following
-the UEFI Platform Initialization Specification cite:[UEFI-PI].
-
-Implementing the following elements is recommended:
-
-* Security Architectural Protocol (EFI_SECURITY_ARCH_PROTOCOL)
-* Security2 Architectural Protocol (EFI_SECURITY2_ARCH_PROTOCOL)footnote:[The
-Security and Security2 Architectural Protocols are overridden by some
-bootloaders (e.g. systemd-boot) to validate EFI binaries that cannot be
-validated against the UEFI security database.]
-
-=== UEFI System Environment and Configuration
-==== Privilege Levels
-===== UEFI Boot in HS-Mode
-===== UEFI Boot in (V)S-Mode
-==== System Volume Format
-==== UEFI Image Format
-===== UEFI Drivers
-===== UEFI Applications
-===== PE/COFF Images
-==== Graphics Output (GOP) Protocol
-==== Address Translation Support
-=== UEFI Boot Services
-==== Memory Map
-==== UEFI Loaded Images
-==== Configuration Tables
+
+The Unified Extensible Firmware Interface (UEFI) specification describes the interface between the OS and the supervisor-mode firmware.
+
+This section defines the BRS-I mandatory and optional UEFI requirements on top of cite:[UEFI], and is optional and recommended for BRS-B. Additional non-normative guidance may be found in the <<uefi-guidance, appendix>>.
+
+A compliant system must:
+
+* Implement a 64-bit UEFI firmware.
+* Meet the 3rd Party UEFI Certificate Authority (CA) requirements on UEFI memory mitigations cite:[MSUefiCaRequirements].
+* Meet the following UEFI memory map requirements:
+** The default memory space attribute must be EFI_MEMORY_WB.
+** Enable address translation.
+*** Any memory space defined by the UEFI memory map must be identity mapped (virtual address equals physical address), although the attributes of certain regions may not have all read, write and execute attributes or be unmarked for purposes of platform protection.
```

Comment:
These items seem to be extracted from ia32 requirements, correct? Shouldn't we feed that directly back into UEFI spec?

Reply:
@andreiw says these requirements are in-flight w/ uefi spec proper to be included. uefi calling conventions sections for risc-v. Approval is supposedly this week (sept 11)